spec/unit/installer/xcode/pods_project_generator_spec.rb

```diff
@@ -267,6 +288,46 @@ class Xcode
             @generator.project.targets.find { |t| t.name == 'CoconutLib-iOS' }.isa.should == 'PBXAggregateTarget'
           end
 
+          it 'creates and links app host with an iOS test native target' do
+            @generator.generate!
+            @generator.project.targets.count.should == 19
+            app_host_target = @generator.project.targets[17]
```

Comment:
can also prefer to do `@generator.project.targets.find { |t| t.name == 'AppHost-iOS-Unit-Tests` instead of the index based finding.

Then just assert its not `nil` (via `should.not.be.nil`)